Blog/views.py

```diff
@@ -18,8 +19,23 @@ def index(request):
 
 
 class PostListView(generic.ListView):
-    model = Post
     paginate_by = 10
+    template_name = 'Blog/post_list.html'
```

Comment:
If I'm not completely mistaken, because of the way Django constructs the template name (see django.views.generic.list.MultipleObjectTemplateResponseMixin#get_template_names)